docs/_includes/input-chips.html

```diff
@@ -0,0 +1,344 @@
+<div id="input-chips">
+    {% include section-header.html name="input-chips" version=page.versions.input-chips %}
+
+    <p>Input chips are creators and managers of <span class="highlight">chip</span> components. They allow user search of existing chip options, additions, and removals of chips. The search/selection follows the pattern of the combobox pattern.</p>
+    
+    <h3>Basic Input Chips</h3>
+    <p>This is the simplest type of implementation and includes only the bare essentials for using the module.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="input-chips">
+                <div class="input-chips__container">
+                    <div class="input-chips__content">
```

Comment:
Is there a reason why we have 3 levels? Can we not simply flatten this by one level?

So like all styles for `.input-chips__container` becomes `.input-chips` style. Then we could rename `input-chips__content` to be `input-chips__container` (I prefer the name input chips container for the container of input chips)

Reply:
Ok, so the other reason why there's that extra layer is because `input-chips` has an attached label and helper text that both need to sit outside the box.

Reply:
Hmm, I think we should look at keeping the label outside the scope of this module, like we do for textbox and combobox. That way developers can position the label wherever they like. Unless I'm missing something?

Reply:
Agreed. Field should be used in that case.